### PR TITLE
Add disabledFeatures config check for hybrid flow configuration

### PR DESCRIPTION
**Pull request by asha15**
### Purpose
$subject

### Related Issues
- https://github.com/wso2/product-is/issues/20812

### Related PRs
- 

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: features/admin.applications.v1/components/forms/inbound-oidc-form.tsx [hunk 1]
```diff
@@ -339,6 +342,16 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
...
 
+    const featureConfig: FeatureAccessConfigInterface = useSelector((state: AppState) => {
+        return state.config?.ui?.features?.applications;
+    });
+
+    const IsHybridFlowAppLevelConfigEnabled: boolean = useMemo(() => {
+        const disabledFeatures: string[] = featureConfig?.disabledFeatures;
+
+        return !disabledFeatures.includes("applications.hybridFlow");
+    }, [ featureConfig ]);
```
Comment (pavinduLakshan) [inline features/admin.applications.v1/components/forms/inbound-oidc-form.tsx:353]: Use the readymade functions in core modules instead.

Ref: https://github.com/wso2/identity-apps/blob/master/docs/FEATURE_FLAGS.md#:~:text=%5D%0A%20%20%7D%0A%7D-,Use%20Feature%20Flags%20in%20Your%20Code,-Finally%2C%20use%20the

**Review by pavinduLakshan — COMMENTED** [review]
Let's fix the lint issues and address the comments.

File: features/admin.applications.v1/components/forms/inbound-oidc-form.tsx (hunk 1 above)
Comment (asha15) [reply features/admin.applications.v1/components/forms/inbound-oidc-form.tsx]: Fixed with: https://github.com/wso2/identity-apps/pull/6709/commits/05f71e055f845e30333310622a50976081a25647

File: features/admin.applications.v1/components/forms/inbound-oidc-form.tsx [hunk 2]
```diff
@@ -533,7 +539,8 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
 
     useEffect(() => {
         setHybridFlowEnableConfig(false);
-        if (selectedGrantTypes?.includes(ApplicationManagementConstants.AUTHORIZATION_CODE_GRANT)) {
+        if (selectedGrantTypes?.includes(ApplicationManagementConstants.AUTHORIZATION_CODE_GRANT)
+            && isFeatureEnabled(applicationFeatureConfig, "applications.hybridFlow")) {
```
Comment (pavinduLakshan) [inline features/admin.applications.v1/components/forms/inbound-oidc-form.tsx:543]:
```suggestion
        if (selectedGrantTypes?.includes(ApplicationManagementConstants.AUTHORIZATION_CODE_GRANT)
            && isFeatureEnabled(applicationFeatureConfig, "applications.hybridFlow")
        ) {
```

**Review by pavinduLakshan — APPROVED** [review]

**Pull request merged**